protocol/contracts/leaser/src/state/config.rs

```diff
@@ -61,21 +62,25 @@ impl Config {
         lease_due_period: Duration,
     ) -> ContractResult<()> {
         Self::STORAGE
-            .update(storage, |mut c| -> ContractResult<Config> {
-                c.lease_interest_rate_margin = lease_interest_rate_margin;
-                c.lease_position_spec = lease_position_spec;
-                c.lease_due_period = lease_due_period;
-                Ok(c)
+            .update(storage, |c| -> ContractResult<Config> {
```

Comment:
I thought you were pushing against explicit types. ;)
```suggestion
            .update(storage, |c| {
```

Reply:
right, this code is not in the scope of the PR ... but I am going to fix it